### ISSUE TITLE
Size of Model

### ISSUE THREAD

**Issue by Jiatttliu** [btoussai/ProbeSDF#6]
Hi, this is a wonderful work! And I want to know how to get the size of the model?

**Comment by btoussai** [comment]
Hi,
The model size will vary on a per-scene basis since it depends on the number of allocated voxels, and is roughly proportional to the surface area of the object. The numbers reported in the paper correspond to the average of the model sizes over all the scenes of the corresponding dataset.

I should make it clear that there is no way to know the size of the model before running the algorithm. That's because the voxels are automatically allocated / pruned during training. Only the final size of the voxels is imposed as a hyper-parameter.

Because of the sparse structure representing the scene with discrete blocks of 4^3 voxels, a lot of them end-up unused but they still take space in VRAM. Thus, the reconstruction algorithm outputs a zip file to avoid storing a bunch of zeros on disk, that would take space but do not correspond to actual parameters.

We report the size of the zip files, which take approximately twice as much space once uncompressed.
You can refer to our previous paper [millimetrichumans](https://projects-morpheo.gitlabpages.inria.fr/millimetrichumans/) if you need more details about the sparse grid.

Best,
Briac

**Comment by Jiatttliu** [comment]
Thanks for your reply!!! And could you tell me which zip file should I download from url:(https://github.com/YoYo000/BlendedMVS). I have download [textured meshes](https://1drv.ms/u/s!Ag8Dbz2Aqc81fkvi2X9Mmzan0FI?e=7x2WoS) but it seem not suit for the repo.

**Comment by btoussai** [comment]
As explained in the [readme file](https://gitlab.inria.fr/projects-morpheo/ProbeSDF/-/blob/main/README.md?ref_type=heads#blendedmvs-dataset) file,
we use the preprocessed images from neus, that you can get [here](https://www.dropbox.com/scl/fo/um3wj3ctiuoottbfmqmgb/ABZRltszDvWHJ824UL6DHw0?rlkey=3vjok0aivnoiaf8z5j6w05k92&e=1&dl=0).

**Comment by Jiatttliu** [comment]
That is right, I have get the RGB and Mask, but I can not find groud truth mesh like `.PLY` file in `dataset_texture_mesh.zip` downloaded from [BlenderMVS](https://1drv.ms/u/s!Ag8Dbz2Aqc81fkvi2X9Mmzan0FI?e=7x2WoS)

**Comment by btoussai** [comment]
For convenience, here are the meshes we used for evaluation:
https://drive.google.com/file/d/1k6TrFiLBp8dWg2E8ur6KbL6R04tNd4Ln/view?usp=sharing

**Comment by Jiatttliu** [comment]
Hi, @btoussai, could you tell me how to get the rendered normal or rendered depth at all viewpoints? I'd like to add some evaluations to our work which need to use the above results in these evaluations.

Best!

**Comment by btoussai** [comment]
For this, I think you can just render the extracted mesh in the original viewpoints.
Using a python blender script, opengl, or open3d are possible options.
